cartridges/custom_sfcc_learning/cartridge/scripts/steps/exportCustomerWishlist.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+var CustomerMgr = require("dw/customer/CustomerMgr");
+var productListHelper = require('lib_productlist/cartridge/scripts/productList/productListHelpers');
+var File = require('dw/io/File');
+var FileWriter = require('dw/io/FileWriter');
+var FileReader = require('dw/io/FileReader');
+
+module.exports = {
+    exportWishlist: function exportWishlist() {
+        var file = new File(File.IMPEX + '/src/CustomerList.xml');
```

Comment:
Could we put this file name and path into a property so it could be configurable?

Reply:
Used Resource.msg() to call the property.